graphql/types/Shipping.graphql

```diff
@@ -9,20 +9,31 @@ type Shipping {
 
 type DeliveryOption {
   id: String!
+  deliveryChannel: String!
   price: Int!
   estimate: String!
   isSelected: Boolean!
 }
 
 type PickupOption {
   id: String!
+  address: Address!
+  deliveryChannel: String!
   price: Int!
   estimate: String!
   isSelected: Boolean!
   friendlyName: String!
   additionalInfo: String!
   storeDistance: Float!
   transitTime: String!
+  businessHours: [BusinessHour!]!
+}
+
+type BusinessHour {
+  dayNumber: String!
+  closed: Boolean!
+  closingTime: String!
+  openingTime: String!
```

Comment:
What do you think about using a `Date` scalar or something like that?

Reply:
I know that it works and should be fine for now, but maybe we could add this scalar in the future to improve type safety

Reply:
`Date` isn't a primitive scalar in graphql

Reply:
sure, but my bad, I wasn't very clear. I mean, probably the backend respects some specific ISO and creating a custom scalar that adheres to the same ISO should be good, right?

Reply:
idk, because graphql can't send to the client an actual `Date` instance, we will need to convert it to string of a number nevertheless. I think using `String` here is good enough, and we can create a `Date` instance in the UI if we need to

Reply:
Hmm, I see. Thanks for the explanation